llm/src/main/java/com/instana/dc/llm/LLMDcUtil.java

```diff
@@ -46,11 +46,27 @@ public class LLMDcUtil {
     public static final String LLM_COST_NAME = "llm.usage.cost";
     public static final String LLM_COST_DESC = "The total cost of watsonx calls by interval";
     public static final String LLM_COST_UNIT = "{cost}";
+
+    public static final String LLM_INPUT_COST_NAME = "llm.usage.input_cost";
+    public static final String LLM_INPUT_COST_DESC = "The input cost of watsonx calls by interval";
```

Comment:
@LakshmiPriyaSujith this maybe a follow up PR, can you replace `watsonx` to `llm` for all the DESC here?

Reply:
I will raise a follow up PR for this.